src/client/components/Signup.test.tsx

```diff
@@ -0,0 +1,43 @@
+import puppeteer from 'puppeteer';
+
+
```

Comment:
Did you ensure that these test fail when the behavior is not exhibited, aka make the sad path in the code (Signup.tsx) and test it out

Reply:
not yet